packages/api-key-stamper/src/__tests__/elliptic-curves-test.ts

```diff
@@ -0,0 +1,55 @@
+import { test, expect } from "@jest/globals";
+import { pointDecode } from "../tink/elliptic_curves";
+import {
+  uint8ArrayFromHexString,
+  uint8ArrayToHexString,
+} from "@turnkey/encoding";
+
+test("pointDecode -> uncompressed invalid", async function () {
```

Comment:
I don't see a test for compressed keys and invalid lengths cases. Let's cover these?

Reply:
I'll add that in!